api/http/routes/email.route.go

```
@@ -22,7 +23,9 @@ func EmailHandler(c *gin.Context) {
 	if err := c.BindJSON(&requestBody); err != nil {
 		c.String(http.StatusBadRequest, err.Error())
 	}
+	ctx := context.WithoutCancel(c.Request.Context())
 	go core.NewCore().EmailUsecase.Send(
```

golangci-lint (2.12.2)

[error] Error return value of `.Send` is not checked

(errcheck)